docs/javascript/development-process.md

```diff
@@ -10,7 +10,7 @@ html_meta:
 
 ## Code Style
 
-Together with [plone.api](https://github.com/plone/plone.api) we developed [code style guidelines](https://github.com/plone/plone.api/blob/master/docs/contribute/conventions.rst), which we are enforcing now for core Plone development.
+Together with [plone.api](https://github.com/plone/plone.api) we developed [Styleguides](https://docs.plone.org/develop/styleguide/), which we are enforcing now for core Plone development.
```

Comment:
Style guide is two words in English. We need to fix all instances of that for Plone 6 docs. I don't have interest in updating Plone 5 docs.

```suggestion
Together with [plone.api](https://github.com/plone/plone.api) we developed [Plone style guides](https://docs.plone.org/develop/styleguide/), which we are enforcing now for core Plone development.
```